chapters/annotations.tex

```diff
@@ -869,6 +869,7 @@ \subsection{Component Instance}\label{component-instance}
 end Placement;
 \end{lstlisting}%
 \annotationindex{Placement}
+If a component has \lstinline!visible! being \lstinline!true! (the default) it can be shown, otherwise it shall not be shown.
```

Comment:
Don't we use the equal sign for this?
```suggestion
If a component has \lstinline!visible = true! (the default) it can be shown, otherwise it shall not be shown.
```